scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -64,7 +64,9 @@ import scala.reflect.runtime.universe._
 /** Convenience functions for creating SCollections. */
 object SCollection {
 
-  /** Create a union of multiple SCollections */
+  /** Create a union of multiple SCollections.
```

Comment:
Use JavaDoc indentation please like the rest of the code base. Also you can link API with `[[ScioContext#unionAll]]` or something like that.